src/diffpy/utils/tools.py

```diff
@@ -131,3 +133,31 @@ def get_package_info(package_names, metadata=None):
         pkg_info.update({package: importlib.metadata.version(package)})
     metadata["package_info"] = pkg_info
     return metadata
+
+
+def compute_mu_using_xraydb(sample_composition, energy, density=None, packing_fraction=1):
```

Comment:
added packing fraction. does this make sense or should we use None instead? I think if we set the default to 1 the code can be more concise

Reply:
I think setting to None is clearer here.   The logic from the perspective of the user is that we either have a packing fraction or we don't.  If it is None, you can set it to 1 in the code.

Reply:
Also, didn't we discuss to change the name to `sample_mass_density`?